src/simulator_core/entities/assets/utils.py

```diff
@@ -65,6 +65,40 @@ def mass_flow_and_temperature_to_heat_demand(
     return mass_flow * (temperature_supply - temperature_return) * float(heat_capacity)
 
 
+def mass_flow_to_volume_flow(
+        mass_flowrate: float,
+        temperature_fluid: float,
+        pandapipes_net: pandapipesNet,
+) -> float:
+    """Calculate the volume flowrate from mass flowrate.
+
+    :param float mass_flowrate: the mass flowrate in kg/s
+    :param float temperature_fluid: the fluid temperature in K
+    :param pandapipesNet net: The pandapipes network used to calculate the specific density.
+    """
+    density_fluid = pandapipes_net.fluid.get_density(temperature_fluid)
+    volume_flowrate = mass_flowrate * 3600 / density_fluid
```

Comment:
Is this correct? You have mass_flowrate in kg/s then you are going to a float but do you want m3/h?

Add the output and its units to the docstring. I see that it is also missing for the other utility function so maybe you can also add it there?

Reply:
yes, I want to have volumentric flowrate (which is m3/h is common)

Reply:
I would prefer to do it in SI (m3/s), and unit conversion will be done at the level where results are presented to the user. But maybe good to make an issue on discussion of internal units and leave it for now.

Reply:
fix to m3/s